src/api/app/views/webui/shared/_buildresult_box.html.haml

```diff
@@ -1,51 +1,68 @@
 :ruby
   index ||= ''
+  project ||= nil
+  package ||= nil
+  bs_request ||= nil
+  buildable = package || project
 
-  # For a request staged in a staging project, we display the Build Results / RPM Lint from the staging project instead
-  is_staged_request = defined?(bs_request) && !bs_request.staging_project_id.nil?
-  project = bs_request.staging_project.name if is_staged_request
+  if buildable
```

Comment:
@saraycp I think you can move the content of this if block inside the next one in line 19

Reply:
@krauselukas updated!